BullsEye/AboutViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AboutViewController.swift
+//  BullsEye
+//
+//  Created by Tim Lauter on 11/2/16.
+//  Copyright © 2016 timlauter. All rights reserved.
+//
+
+import UIKit
+
+class AboutViewController: UIViewController {
+    @IBOutlet weak var webView: UIWebView!
+    @IBAction func close() {
+        dismiss(animated: true, completion: nil)
+    }
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        if let url = Bundle.main.url(forResource: "BullsEye", withExtension: "html") {
+           
+            if let htmlData = try? Data(contentsOf: url) {
```

Comment:
You have external resources defined in your `Downloads` folder which we don't get when you clone the project. Those files should be directly copied into your project resources.